client/src/components/EditProfile/EditForm/useStyles.ts

```diff
@@ -0,0 +1,20 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  boxButton: {
+    textAlign: 'center',
+    paddingTop: '10px',
+  },
+  typography: {
+    color: '#000',
```

Comment:
Have a look at typography's [`color` prop](https://v4.mui.com/api/typography/#props), would it work to use one of those instead? We should try to avoid setting colors in css and leverage the mui theme as much as possible.